Api/app/controllers/requests_controller.rb

```diff
@@ -1,7 +1,7 @@
 class RequestsController < ApplicationController
 
   def index
-    render json: current_user.requester_request + current_user.requested_request
```

Comment:
@MariethPerez is there any reason why we were retrieving the requests like this?